events.md

```diff
@@ -0,0 +1,271 @@
+-   `Organization`: `Organization` - The Organization model tied to this event. Check `server/models/Organization.js` for available properties.
+
+# Core Events
+
+The following events exist without any skills installed.
+
+TODO: add additional core events (location-schedule, teammate-location-schedule, etc)
+
+-   `did-signup` - When a guest "opts into" a location. Check `ctx.status === 'online'` to know if they signed up by joining the wifi.
+-   `did-enter` - When a guest returns and their phone hits the wifi.
+-   `did-leave` - Triggered an hour after a guest leaves the wifi.
+-   `did-message` - A guest has sent a text to Sprucebot.
+-   `did-add-device` - When a guest adds a new device to a location (.e.g. laptop or tablet).
+-   `did-update-profile` - When any user updates their first or last name.
+-   `did-opt-out` - When any guest opts out of a location. By now you have already lost access to their meta data.
+-   `did-rejoin` - They had, at one time, opted out. But, now they have remotely opted back in.
+-   `will-send-training` - Sprucebot has made the decision that now is the perfect time to send training material.
+-   `was-installed` - When the skill is first installed to a location.
+
+# Emitting events
+
+My favorite part of the event system is that installing new skills introduces new events into the platform. This allows you to build your motion sensor skill, connect it to a RaspberryPi, emit a `motion-sensor:did-detect-motion` event, and give other skills the ability to build entire Experiences triggered by motion.
+
+But, that's not the only reason you want to emit an event in your skill. You may actually want to utilize another skill's CRUD event to save or update a setting.
+
+There are actually 4 ways you can emit an event:
+
+1. To all skills enabled at a location
+    1. Great for real world events that take place at a location that has local effects, like a motion sensor
+    2. The event name will start with your skill's slug
+2. To a specific skill enabled at a location
+    1. Used for CRUD events, e.g. `booking:book-appointment`
+    2. The event name starts with the target skill's slug
+3. To all skills installed at an organization
+    1. For real world events that have organization wide impacts, e.g. `booking:did-delete-service`
+    2. The event name will start with your skill's slug
+4. To a specific skill installed at an organization
+    1. Used for CRUD events that apply to an entire org, e.g. `booking:create-service`
+    2. The event name starts with the target skill's slug
+
+## Step 1: Event contracts
+
+Just like you did for events you want to subscribe to, you have to define the skills you will be emitting.
+
+### Example
+
+```js
+// config/default.js
+eventContract: {
+  events: {
+    'my-awesome-skill:did-trigger-motion-sensor': { // the slug of your skill and then the name of your event
+        publish: true // you are publishing this event
```

Comment:
We don't currently support `publish` - this (as it is right now) would actually subscribe to the event.

We should get this functionality in though along w/ adding types to the event contracts